commit 3edbed0dd13f290083671631af6b39c10a3d1a7e
Author: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date:   Sat Dec 29 13:57:11 2018 +0100

    ARM: dts: meson8m2: mxiii-plus: mark the SD card detection GPIO active-low
    
    [ Upstream commit 8615f5596335db0978cea593dcd0070dc5f8b116 ]
    
    After commit 89a5e15bcba87d ("gpio/mmc/of: Respect polarity in the device
    tree") SD cards are not detected anymore.
    
    The CD GPIO is "active low" on the MXIII-Plus. The MMC dt-bindings
    specify: "[...] using the "cd-inverted" property means, that the CD line
    is active high, i.e. it is high, when a card is inserted".
    
    Fix the description of the SD card by marking it as GPIO_ACTIVE_LOW and
    drop the "cd-inverted" property. This makes the definition consistent
    with the existing dt-bindings and fixes the check whether an SD card is
    inserted.
    
    Fixes: 35ee52bea66c74 ("ARM: dts: meson8m2: add support for the Tronsmart MXIII Plus")
    Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
    Reviewed-by: Linus Walleij <linus.walleij@linaro.org>
    Signed-off-by: Kevin Hilman <khilman@baylibre.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/meson8m2-mxiii-plus.dts b/arch/arm/boot/dts/meson8m2-mxiii-plus.dts
index f5853610b20b..6ac02beb5fa7 100644
--- a/arch/arm/boot/dts/meson8m2-mxiii-plus.dts
+++ b/arch/arm/boot/dts/meson8m2-mxiii-plus.dts
@@ -206,8 +206,7 @@
 		cap-sd-highspeed;
 		disable-wp;
 
-		cd-gpios = <&gpio CARD_6 GPIO_ACTIVE_HIGH>;
-		cd-inverted;
+		cd-gpios = <&gpio CARD_6 GPIO_ACTIVE_LOW>;
 
 		vmmc-supply = <&vcc_3v3>;
 	};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1066_3edbed0dd13f290083671631af6b39c10a3d1a7e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1066_3edbed0dd13f290083671631af6b39c10a3d1a7e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
